src/network/core/game_info.h

```diff
@@ -50,6 +89,19 @@ struct NetworkGameInfo : NetworkServerGameInfo {
 	byte map_set;                                   ///< Graphical set
 };
 
-const char * GetNetworkRevisionString();
+extern NetworkServerGameInfo _network_game_info;
+
+const char *GetNetworkRevisionString();
```

Comment:
I'm not sure whether version handling belongs with the game info. At some level it makes sense, on the other hand it's used more generic in other places of the network code.

Reply:
I had the same consideration, but in the end, I decided it is at least not unreasonable to be here, and as it was already here, not to move it.